crates/burn-tensor/src/tensor/api/numeric.rs

```diff
@@ -70,6 +70,14 @@ where
         Self::new(K::remainder_scalar::<E>(self.primitive, other))
     }
 
+    /// Applies element wise the remainder operation with a scalar.
+    ///
+    /// `y = x2 % x1`
+    #[allow(clippy::should_implement_trait)]
```

Comment:
Any reason for this attribute?

Reply:
Other methods in this fn needs them to avoid clippy warning like 

> method `add` can be confused for the standard trait method `std::ops::Add::add`

But I think that warning would only occur if we are using name `rem` in our case. So you are correct! Will remove this attribute for both `remainder` and `remainder_scalar`